barretenberg/cpp/scripts/bb-tests.sh

```diff
@@ -11,8 +11,18 @@ IMAGE_URI=$(calculate_image_uri $REPOSITORY)
 retry docker pull $IMAGE_URI
 
 TESTS=(
-  client_ivc_tests
+  flavor_tests
```

Comment:
Did this so that when we run CI first we run the less complex tests first, so if we segfault or fail an assertion we can see the failure farther upstream.